suites/quincy/rbd/tier-2_rbd_regression.yaml

```diff
@@ -160,3 +160,15 @@ tests:
       module: rbd_snapshot_rename_advanced.py
       name: Test Snapshot Rename with clone operations in progress
       polarion-id: CEPH-9836
+
+  - test:
+      desc: Trying to delete a protected snapshot should fail
```

Comment:
please add Negative test in the description

Reply:
done